crates/ironrdp-bench/benches/bench.rs

```diff
@@ -9,29 +9,40 @@ use ironrdp_server::BitmapUpdate;
 pub fn rfx_enc_tile_bench(c: &mut Criterion) {
     let quant = rfx::Quant::default();
     let algo = rfx::EntropyAlgorithm::Rlgr3;
+
+    const WIDTH: NonZeroU16 = NonZeroU16::new(64).expect("value is guaranteed to be non-zero");
+    const HEIGHT: NonZeroU16 = NonZeroU16::new(64).expect("value is guaranteed to be non-zero");
+    const STRIDE: NonZeroUsize = NonZeroUsize::new(64 * 4).expect("value is guaranteed to be non-zero");
+
     let bitmap = BitmapUpdate {
         x: 0,
         y: 0,
-        width: NonZero::new(64).unwrap(),
-        height: NonZero::new(64).unwrap(),
+        width: WIDTH,
+        height: HEIGHT,
         format: ironrdp_server::PixelFormat::ARgb32,
         data: vec![0; 64 * 64 * 4].into(),
-        stride: NonZero::new(64 * 4).unwrap(),
+        stride: STRIDE,
     };
     c.bench_function("rfx_enc_tile", |b| b.iter(|| rfx_enc_tile(&bitmap, &quant, algo, 0, 0)));
 }
 
 pub fn rfx_enc_bench(c: &mut Criterion) {
     let quant = rfx::Quant::default();
     let algo = rfx::EntropyAlgorithm::Rlgr3;
+
+    const WIDTH: NonZeroU16 = NonZeroU16::new(2048).expect("value is guaranteed to be non-zero");
+    const HEIGHT: NonZeroU16 = NonZeroU16::new(2048).expect("value is guaranteed to be non-zero");
+    // QUESTION: It looks like we have a bug here, don't we? The stride value should be 2048 * 4.
```

Comment:
Hi @elmarco!  Could you help with clarifying this one? It seems it has been modified by you.